app/src/main/java/org/fossasia/openevent/fragments/LocationsFragment.java

```diff
@@ -47,15 +48,27 @@
 
     private SearchView searchView;
 
+    private TextView noLocations;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'noLocations' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406594558/issues/source?bid=3233850&fileBranchId=3233896#l51)